pyrightconfig.stricter.json

```diff
@@ -71,6 +71,7 @@
         "stubs/SQLAlchemy",
         "stubs/stripe",
         "stubs/tqdm",
+        "stubs/translationstring",
```

Comment:
Doesn't look like we need to exclude these stubs from the stricter pyright settings, since all parameters and return types appear to already be annotated :)

```suggestion
```